benchmarks/ALCE/ELI5/eli5.py

```diff
@@ -0,0 +1,176 @@
+import json
+import os
+import time
+
+from datasets import Dataset
+from tqdm import tqdm
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+import rageval as rl
+from rageval.models.openai import OpenAILLM
+from utils import make_demo, create_eli5_eval_dataset
+
+
+class ELI5:
+
+    def __init__(self, args):
+        self.args = args
+
+        self.model = None
+        self.tokenizer = None
+        self.eval_data = None
+        self.prompt = None
+        self.name = None
+        self.result_path = None
+
+    def generate_init(self):
+        print("-" * 10 + "Loading model" + "-" * 10)
+        if "gpt-3.5-turbo" in self.args.model:
+            os.environ["OPENAI_API_KEY"] = self.args.api_key
+            self.model = OpenAILLM(
+                model=self.args.model,
+                _api_key_env_var="OPENAI_API_KEY",
+                max_tokens=self.args.max_tokens,
+                temperature=self.args.temperature,
+                top_p=self.args.top_p
+            )
+        else:
+            self.tokenizer = AutoTokenizer.from_pretrained(self.args.model, use_fast=False)
+            self.model = AutoModelForCausalLM.from_pretrained(
+                self.args.model,
+                device_map='auto'
+            )
+        print("-" * 10 + "Finish loading model" + "-" * 10)
+
+        print("-" * 10 + "Loading dataset" + "-" * 10)
+        if self.args.dataset == "bm25":
```

Comment:
are there two version of validation dataset in original ALCE benchmark?

Reply:
Yes, ALCE's ELI5 data set provides two versions: `bm25` and `oracle`. The docs provided by the two datasets are different, and the docs of the latter are of higher quality.